plugins/autolabor/autohauler.cpp

```diff
@@ -583,33 +457,24 @@ DFhackCExport command_result plugin_onupdate ( color_ostream &out )
         {
             dwarf_info[dwarf].state = IDLE;
         }
-        // If it gets to this point we look at the task and assign either BUSY or OTHER
         else
         {
             int job = dwarfs[dwarf]->job.current_job->job_type;
             if (job >= 0 && size_t(job) < ARRAY_COUNT(dwarf_states))
                 dwarf_info[dwarf].state = dwarf_states[job];
             else
             {
-                // Warn the console that the dwarf has an unregistered labor, default to BUSY
-                out.print("Dwarf %i \"%s\" has unknown job %i\n", dwarf, dwarfs[dwarf]->name.first_name.c_str(), job);
-                dwarf_info[dwarf].state = BUSY;
+                WARN(cycle, out).print("Dwarf %i \"%s\" has unknown job %i\n", dwarf, dwarfs[dwarf]->name.first_name.c_str(), job);
+                dwarf_info[dwarf].state = OTHER;
             }
         }
 
-        // Debug: Output dwarf job and state data
-        if(print_debug)
-            out.print("Dwarf %i %s State: %i\n", dwarf, dwarfs[dwarf]->name.first_name.c_str(),
-                      dwarf_info[dwarf].state);
-
-        // Increment corresponding labor in default_labor_infos struct
         state_count[dwarf_info[dwarf].state]++;
 
+        INFO(cycle, out).print("Dwarf %i \"%s\": state %s\n",
```

Comment:
This will be output by default. Is that what you want? It seems like it might get a bit chatty.

Reply:
oh, yeah, that one probably should be trace, thanks, will fix